packages/@lwc/engine-dom/README.md

```diff
@@ -1 +1,149 @@
 # @lwc/engine-dom
+
+This package can be used to render LWC components as DOM elements in a DOM environment.
+
+Usage of internal APIs are prevented by the compiler and are therefore not documented here.
+
+## Supported APIs
+
+This package supports the following APIs.
+
+### createElement()
+
+This function creates an LWC component, given a tag name and an LWC constructor.
+
+```js
+import { createElement } from '@lwc/engine-dom';
+import LightningHello from 'lightning/hello';
+
+const element = createElement('lightning-hello', { is: LightningHello });
+
+document.body.appendChild(element);
+```
+
+### LightningElement
```

Comment:
I don't think we should list those APIs that are exported out of `@lwc/engine-core` here. Let's move the `LightningElement`, `@api`, ... APIs to the `@lwc/engine-core` package README.

Reply:
Yeah, I initially had them in engine-core but moved them out because that's not where developers should be importing them from, but I guess the README is not where they should be looking for our API documentation. I will move them back.